AnnoDesigner.Core/Models/IBounded.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System.Windows;
+
+namespace AnnoDesigner.Core.Models
+{
+    public interface IBounded
```

Comment:
I think `IBounded` is slightly too generic in this context. Bounded could also relate to other "shapes", whereas this use relies on a rectangular set of bounds. Maybe renaming to something like `IHasRectangularBounds`, `IBoundedRect`, `IRectangular`, `IRectangularBounds`? I'm really not sure 😃 .

Reply:
I don't know, `IBounded` seems fine to me, since ti doesn't say anything about it needing to be a rectangle, just that is is bounded. But it's just a naming so I don't really care. These examples just seem very verbose to me and don't roll of tongue very well.

Reply:
Fair enough.